### DISCUSSION TITLE
【疑问】所以绳网升级机制是什么呢？

### DISCUSSION THREAD

**Discussion by xing-qiong** [share121/inter-knot#3214] [其他]
为什么发帖回帖都不见经验增长啊<img src="https://bbs-static.miyoushe.com/static/2024/06/28/5e73914d0c2348d5cb376a00ab82aad7_180874175435333649.png" alt="松雀-我吗" width="75" >

**Comment by Alsuhold** [comment]
这玩意好像是注册时随机固定的

**Comment by liaoli-hero** [comment]
不清楚，反正我还差2000经验升级

**Comment by shan-dgha** [comment]
有大佬在别的贴子解释了，但我看不懂

**Reply by Alsuhold** [reply]
大概就是你在github上公布的项目数量<img src="https://img-static.mihoyo.com/communityweb/upload/9249bd36125b953ddd445b19151427d6.png" alt="安比 疑惑" width="75" >

**Comment by chaojituentuenshu** [comment]
多搞几个存储库就可以了，有多少个就多少级

**Reply by isle200inf** [reply]
20个空库<img src="https://upload-bbs.miyoushe.com/upload/2023/03/09/3d389fd2885b008e46d843c552c02bf3_5073438697399152551.png" alt="开拓者 吃瓜脸" width="75" >

**Reply by Enigma-Soul** [reply]
666 ，起码写点东西吧，要不是这是对开发者的侮辱

**Reply by isle200inf** [reply]
> 666 ，起码写点东西吧，要不是这是对开发者的侮辱

智齿
